components/helminstaller/README.md

```diff
@@ -0,0 +1,39 @@
+# Example helminstaller
+
+This folder contains an example how to build an installer for the [toi](../../docs/reference/ocm_toi-bootstrapping.md) of the ocm CLI. It also demonstrates how to use OCM with building multiarch-images. It contains
+an `executorspec.yaml` as interface to the toi installer.
+
+
+# Building
+
+You can use `make` to build this component. You will have to adjust the variables at the top of the [makefile](Makefile) to your environment (at least `OCMREPO`). By default all artifacts are built in the `gen` folder of this project.
+
+The main targets are:
+
+* `make ca`: builds a component archive
+* `make ctf`: builds a common transport archive
+* `make push`: stores the component version in an OCI registry
+* `make transport`: transfers the component version between two registries
+* `make descriptor`: displays the component descriptor
+* `make describe`: displays the component and it dependencies in a tree structure
+
+
+# Usage
```

Comment:
Maybe we shoud a ink to the cmds/hemlinstaller README. Here the config is explaines. (some docu added with the toi branch in mandelsoft). Should be merge it? (https://github.com/open-component-model/ocm/pull/305)